src/chat-bubble/chat-bubble-base.style.ts

```diff
@@ -0,0 +1,81 @@
+import { css, unsafeCSS } from "lit";
+import { BaseParts } from "./constants";
+
+const root = unsafeCSS(BaseParts.ROOT);
+const footer = unsafeCSS(BaseParts.FOOTER);
+const body = unsafeCSS(BaseParts.BODY);
```

Comment:
I’m open to the idea of using predefined class names as a convention for styling. However, let us address this in a separate discussion. If we adopt this convention across the codebase, we’ll need to create an issue tracker to update all components to follow this approach systematically. Otherwise, we risk introducing inconsistencies in how similar styling problems are addressed across the codebase.

Reply:
Majid and I thoroughly discussed this topic and reached an understanding about the tradeoffs and potential issues it might raise. Therefore, we will not be accepting this pattern.